test/integration/stats_integration_test.cc

```diff
@@ -209,6 +209,7 @@ TEST_P(ClusterMemoryTestRunner, MemoryLargeClusterSizeWithStats) {
   // 2019/06/17  7243     49412       49700   macros for exact/upper-bound memory checks
   // 2019/06/29  7364     45685       46000   combine 2 levels of stat ref-counting into 1
   // 2019/06/30  7428     42742       43000   remove stats multiple inheritance, inline HeapStatData
+  // 2019/07/06  7477     42742       43000   fork gauge representation to drop pending_increment_
```

Comment:
Q: Wouldn't you expect this to drop the size? Or is this lost in padding somewhere?

Reply:
Yes I did expect that. I confirmed in the debugger that sizeof(GaugeImpl) is now 40 whereas it used to be 48. I have two theories:

1. tcmalloc uses a common size bucket for 40-byte objects and 48-byte objects.
2. there are not enough Gauges per cluster to impact the overall allocation size per cluster.

I could easily confirm these theories, except my workstation crashed with a kernel panic and won't come up now :). I am currently waiting for the 4 winds to come and make that problem go away. Or I may just bring up an environment in a VM to see what's going on.

Reply:
My guess is tcmalloc allocates a 64-byte chunk. It's not a big deal if you want to investigate later. I will approve so you can decide.